LongestName/script.js

```diff
@@ -0,0 +1,25 @@
+let name1 = "Chris";
+let name2 = "Jacob";
+let name3 = "Janok";
+
+let length1 = name1.length;
+let length2 = name2.length;
+let length3 = name3.length;
```

Comment:
Even though making these `lengthx` variables makes the lines of code below look shorter, this could be dangerous, especially when using `let`.  If one of the previous names is updated (because `let` will allow this), then these `lengthx` variables will still refer to the initial value that it was assigned.

```
> let name = 'AJ'
undefined
> name
'AJ'
> name.length
2
> len = name.length
2
> name = 'AJay'
'AJay'
> name.length
4
> len
2
```

Using `const` prevents that situation because that variable can't be re-assigned a new value:

```
> const name = 'AJ'
undefined
> name.length
2
> len = name.length
2
> name = 'AJay'
Uncaught TypeError: Assignment to constant variable.
```

However, I would suggest just using `namex.length`. It's not atrociously longer. When you get into very deeply nested values with multiple levels of dot notation, it could be better used then.